src/Core/Util/String/StringModifierInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Util\String;
+
+/**
+ * Defines reusable methods for strings modifications.
+ */
+interface StringModifierInterface
```

Comment:
Maybe too much 😄 do we really expect to have several implementations for this ?

Reply:
the method `splitByCamelCase` was very useful in my case - its a low chance that someone will need it. Main reason I split the functionality due to I could easily apply tests for it, I feel more secure with tests applied here. 

Anyway, in my opinion since it is in `util` folder I would keep it here :slightly_smiling_face:

Reply:
The code `splitByCamelCase` is very good, I was wondering if the interface was too much

But alright 👍 I trust you on this